src/Models/Company.php

```diff
@@ -0,0 +1,352 @@
+    protected $cover;
+
+    /**
+     * Get the company information.
+     *
+     * @var string
+     */
+    protected $information;
+
+    /**
+     * Get the company rules.
+     *
+     * @var string
+     */
+    protected $rules;
+
+    /**
+     * Get the company requirements.
+     *
+     * @var string
+     */
+    protected $requirements;
+
+    /**
+     * Get the companies website.
+     *
+     * @var string
+     */
+    protected $website;
+
+    /**
+     * Get the companies social information.
+     *
+     * @var \TruckersMP\Models\Social
+     */
+    protected $social;
+
+    /**
+     * Get the games the company supports.
+     *
+     * @var \TruckersMP\Models\Game
+     */
+    protected $games;
+
+    /**
+     * The number of members in the company.
+     *
+     * @var int
+     */
+    protected $membersCount;
+
+    /**
+     * The recruitment status of the company.
+     *
+     * @var string
+     */
+    protected $recruitment;
+
+    /**
+     * The language of the company.
+     *
+     * @var string
+     */
+    protected $language;
+
+    /**
+     * If the company is verified.
+     *
+     * @var bool
+     */
+    protected $verified;
+
+    /**
+     * The date and time the company was created.
+     *
+     * @var \Carbon\Carbon
+     */
+    protected $createdAt;
+
+    /**
+     * Create a new Company instance.
+     *
+     * @param array $company
+     *
+     * @throws \Exception
+     */
+    public function __construct(array $company)
+    {
+        $this->id = $company['id'];
+        $this->name = $company['name'];
+        $this->ownerId = $company['owner_id'];
+        $this->ownerName = $company['owner_username'];
+        $this->slogan = $company['slogan'];
+        $this->tag = $company['tag'];
+
+        if (array_key_exists('logo', $company)) {
+            $this->logo = $company['logo'];
+        }
+
+        if (array_key_exists('cover', $company)) {
+            $this->cover = $company['cover'];
+        }
+
+        if (array_key_exists('information', $company)) {
+            $this->information = $company['information'];
+        }
+
+        if (array_key_exists('rules', $company)) {
+            $this->rules = $company['rules'];
+        }
+
+        if (array_key_exists('requirements', $company)) {
+            $this->requirements = $company['requirements'];
+        }
+
+        $this->website = $company['website'];
+
+        $this->social = new Social(
+            $company['socials']['twitter'],
+            $company['socials']['facebook'],
+            $company['socials']['twitch'],
+            $company['socials']['playstv'],
+            $company['socials']['discord'],
+            $company['socials']['youtube']
+        );
+
+        $this->games = new Game(
+            $company['games']['ats'],
+            $company['games']['ets']
+        );
+
+        $this->membersCount = intval($company['members_count']);
+        $this->recruitment = $company['recruitment'];
```

Comment:
I would suggest adding constants for this attribute which are `'Close'` and `'Open'`. Then we could also add methods to check whether the recruitment is closed or not.